src/main/resources/assets/jetorches/lang/lol_us.lang

```diff
@@ -0,0 +1,17 @@
+tab.jetorches.name=Jus Enuf Burny Stikz
```

Comment:
The mod name should remain the same here

Reply:
then why localize it :thonk: